package.json

```diff
@@ -20,7 +20,7 @@
     "prettier-write": "prettier --write",
     "prettier-project": "yarn run prettier-write 'src/**/*.{ts,tsx}'",
     "stylelint-project": " stylelint -- 'src/**/*.{ts,tsx}'",
-    "ci": "yarn type-check && yarn lint && yarn prettier-project -l && yarn test:ci",
```

Comment:
@orta - noticed that `yarn prettier-project -l`, when writing on circle (and thus attempting to write to disk) will error out with an unhelpful error code 1. I was able to notice this due to a file buried in the tree that was presumably committed using `--no-verify`; when running prettier locally and committing those changes everything passed just fine. Any reason this was added to the CI step?

Reply:
That's the command for the CI version of prettier, it lists if you've not shipped prettier'd code - it's just a normal CI check to keep people shipping the right code - yeah, it's STDOUT output is real bad though

I'm not attached to it, it's easy to run prettier across an entire project if it becomes inconsistent eventually

Reply:
Ah, that was def confusing (I should have checked to see what `-l` did).

Readded -- best to have this stuff in place so that things are caught early.